drivers/ieee802154/ieee802154_cc13xx_cc26xx.c

```diff
@@ -155,42 +155,36 @@ static inline int ieee802154_cc13xx_cc26xx_channel_to_frequency(
 	__ASSERT_NO_MSG(frequency != NULL);
 	__ASSERT_NO_MSG(fractFreq != NULL);
 
-	if (channel >= IEEE802154_2_4_GHZ_CHANNEL_MIN
-		&& channel <= IEEE802154_2_4_GHZ_CHANNEL_MAX) {
-		*frequency = 2405 + 5 * (channel - IEEE802154_2_4_GHZ_CHANNEL_MIN);
+	/* See IEEE 802.15.4-2020, section 10.1.3.3. */
+	if (channel >= 11 && channel <= 26) {
+		*frequency = 2405 + 5 * (channel - 11);
 		*fractFreq = 0;
+		return 0;
 	} else {
+		/* TODO: Support sub-GHz for CC13xx rather than having separate drivers */
 		*frequency = 0;
 		*fractFreq = 0;
-		return -EINVAL;
+		return channel < 11 ? -ENOTSUP : -EINVAL;
 	}
-
-	return 0;
```

Comment:
Looks like these changes are not covered by the commit message and probably belong in the next commit, (similar changes https://github.com/zephyrproject-rtos/zephyr/pull/62413/commits/c5684a43c04945d19d5dedbac099e402a3b6e194#diff-8278a3f5641432256702d96822ad636a501df47e873791c58ccdd79b77cad1bc). Cleaner would be to move the removal of `IEEE802154_*_CHANNEL_MAX`/`IEEE802154_*_CHANNEL_MIN` to own commit as they are only used in the _cc13xx_cc26xx_ drivers.

Reply:
@fgrandel Stop to mark resolved unresolved conversations. You are not doing your patches and commits properly. This is not how patches submitted by a maintainer should look. I am clearly showing you a problem in your PR that is still unresolved. And no, you cannot ignore them, because you are the maintainer of that area. These are basic things you learn as a contributor and collaborator: to break your work into smaller, self-contained changes, to have pre-work patches before major changes, and to format and describe them in a way that can be followed just by looking at the git diffs and logs.

Reply:
@fgrandel this request seems to be valid, please just move this hunk to the next commit and let's move on with this.

@jfischer-no Please slow down and take a breath. This is the longest comment I have ever seen about a misplaced hunk in a commit.

Reply:
> @jfischer-no Please slow down and take a breath. This is the longest comment I have ever seen about a misplaced hunk in a commit.

I did not even speed up to slow down. My comment about the inability to submit a reasonable set of patches, compared to the long novellas without significant information content that you otherwise find here, is very concise.

Reply:
@nashif 

> this request seems to be valid, please just move this hunk to the next commit and let's move on with this.

True. Fixed.